boards/xtensa/up_squared_adsp/CMakeLists.txt

```diff
@@ -14,17 +14,17 @@ set(sof_module ${PROJECT_SOURCE_DIR}/../modules/audio/sof)
 
 add_custom_target(
   process_elf
-  DEPENDS sof_base_module
+  DEPENDS base_module
   DEPENDS ${ZEPHYR_FINAL_EXECUTABLE}
-  COMMAND ${CMAKE_OBJCOPY} --dump-section .data=mod-apl.bin ${CMAKE_BINARY_DIR}/modules/sof/libsof_base_module.a
+  COMMAND ${CMAKE_OBJCOPY} --dump-section .data=mod-apl.bin ${CMAKE_BINARY_DIR}/zephyr/boards/xtensa/up_squared_adsp/bootloader/libbase_module.a
```

Comment:
Maybe change `up_squared_adsp` to the board variable? Will make it easier to copy this to another board config in the future.

Reply:
I will try to use BOARD variable